ansible/deploy_commcarehq.yml

```diff
@@ -5,7 +5,7 @@
   # - celery
   # - proxy ?? maybe
   # - pillowtop
-  hosts: all
+  hosts: all:!riakcs:!stanchion
```

Comment:
Don't put localsettings on riakcs servers. We should add more host groups here (or only list host groups where it makes sense to have localsettings). `hqes2` and `hqes3` failed when I ran this against prod, and I assume that's fine and normal.